src/WSRequestHandler_Scenes.cpp

```diff
@@ -148,3 +148,119 @@ RpcResponse WSRequestHandler::ReorderSceneItems(const RpcRequest& request) {
 
 	return request.success();
 }
+
+/**
+ * Set a scene to use a specific transition override.
+ *
+ * @param {String} `sceneName` Name of the scene to switch to.
+ * @param {String} `transitionName` Name of the transition to use.
+ * @param {int (Optional)} `transitionDuration` Duration in milliseconds of the transition if transition is not fixed. Defaults 300 if there is no current override and this value is not given.
+ *
+ * @api requests
+ * @name SetSceneTransitionOverride
+ * @category scenes
+ * @since 4.9.0
+ */
+RpcResponse WSRequestHandler::SetSceneTransitionOverride(const RpcRequest& request) {
+	if (!request.hasField("sceneName") || !request.hasField("transitionName")) {
+		return request.failed("missing request parameters");
+	}
+
+	QString sceneName = obs_data_get_string(request.parameters(), "sceneName");
+	OBSSourceAutoRelease source = obs_get_source_by_name(sceneName.toUtf8());
+	if (!source) {
+		return request.failed("requested scene does not exist");
+	} else if (!(obs_source_get_type(source) == OBS_SOURCE_TYPE_SCENE)) {
```

Comment:
`else if` is not necessary here. If the case above matches, it will exit the function.